types/result.go

```diff
@@ -249,6 +250,27 @@ func (r TxResponse) Empty() bool {
 	return r.TxHash == "" && r.Logs == nil
 }
 
+// SearchTxsResult defines a structure for querying txs pageable
+type SearchTxsResult struct {
+	TotalCount int          `json:"total_count"` // Count of all txs
+	Count      int          `json:"count"`       // Count of txs in current page
+	PageNumber int          `json:"page_number"` // Index of current page, start from 1
+	PageCount  int          `json:"page_count"`  // Count of all pages
```

Comment:
Just a small remark:

```suggestion
	PageTotal  int          `json:"page_total"`  // Total count of pages
```

`count` is used above to indicate the number of items in the current page, whilst `total` represents items grand total. So I thought we could use a consistent `count` vs `total` nomenclature. Not a blocker though.

Reply:
Nah, I think those two are fine

Reply:
Done! @alessio